examples/hello/pubspec.yaml

```diff
@@ -15,3 +15,7 @@ dev_dependencies:
   http: ^0.12.0
   test: ^1.15.0
   test_process: ^1.0.0
+
+dependency_overrides:
```

Comment:
Even though it's part of the repo, I'd prefer the example to remain standalone. Let's discuss offline before going further. Thanks!

Reply:
ack